backend/ml/outputs.py

```diff
@@ -74,14 +78,24 @@ def save_tournesol_scores(poll):
                 None,
             )
         else:
-            entity.tournesol_score = 10 * sum(
+            tournesol_score = 10 * sum(
                 criterion.score for criterion in entity.criteria_scores
             )
+
+        entity.tournesol_score = tournesol_score
         entities.append(entity)
 
+        if entity.single_poll_ratings:
+            entity_poll_rating = entity.single_poll_ratings[0]
```

Comment:
Note sure that's helpful, but we could use `.get()` to ensure that a single item is prefetched:
```suggestion
            entity_poll_rating = entity.single_poll_ratings.get()
```

Reply:
At this point in the code `single_poll_ratings` is a Python list. Calling `.get()` will not work here.